# SPDX-License-Identifier: Apache-2.0
# Copyright (C) 2025 Intel Corporation
adbscan_sub_node:
  ros__parameters:
    Lidar_type: 3D
    Lidar_topic: /scan
    Verbose: True
    subsample_ratio: 20.0
    x_filter_back: 4.0
    y_filter_left: 10.0
    y_filter_right: -20.0
    z_filter: -1.3
    Z_based_ground_removal: 1.0
    base: 3.4694
    coeff_1: -0.0335
    coeff_2: 0.00015124
    scale_factor: 0.7
    oneapi_library: oneapi_kdtree
    benchmark_number_of_frames: 1
